src/callbacks_step/analysis.jl

```diff
@@ -310,7 +310,7 @@ function (analysis_callback::AnalysisCallback)(u_ode, du_ode, integrator, semi)
         mpi_println("                 " * "              " *
                     "               " *
                     " PID:            " * @sprintf("%10.8e s", performance_index))
-        mpi_println(" #DOFs per field:" * @sprintf("% 14d", ndofs(semi)) *
+        mpi_println(" #DOFs per field:" * @sprintf("% 14d", ndofsglobal(semi)) *
```

Comment:
Please check (if not yet done) that all other values that are printed by the analysis callback are also for the global problem and not rank local, otherwise people (especially me 😅) will get confused